ReactiveCocoaTests/Swift/PropertySpec.swift

```diff
@@ -79,6 +79,42 @@ class PropertySpec: QuickSpec {
 				expect(signalCompleted).to(beTruthy())
 			}
 
+			it("should swap the value atomically") {
+				let property = MutableProperty(initialPropertyValue)
+
+				expect(property.swap(subsequentPropertyValue)).to(equal(initialPropertyValue))
```

Comment:
This can be `expect(property.swap(subsequentePropertyValue) == initialPropertyValue`